marlowe-playground-client/src/Blockly.purs

```diff
@@ -149,6 +153,12 @@ render = runFn1 render_
 getBlockById :: Workspace -> String -> Maybe Block
 getBlockById = runFn4 getBlockById_ Just Nothing
 
+workspaceXML :: Blockly -> Workspace -> String
+workspaceXML = runFn2 workspaceXML_
+
+loadWorkspace :: forall r. Blockly -> (STRef r Workspace) -> String -> ST r Unit
+loadWorkspace = runFn3 loadWorkspace_
```

Comment:
I now export the constructor (well actually use the newtype instance) because the gist stuff deals directly with strings rather than `Foreign` or anything else, so I need a way to unwrap it.